lapis/job.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, resources: dict, used_resources: dict, in_queue_since: float
         """
         self.resources = resources
         self.used_resources = used_resources
+        for key in used_resources:
+            if key not in resources:
+                logging.getLogger("implementation")\
+                    .info("job uses different resources than specified, added",
```

Comment:
If you put the ``.info(`` on the ``logging.GetLogger ...`` line, then you can put the arguments on the next lines without a literal line break.

Reply:
Just created #24 that will take care on this.